lib/Controller/RoomController.php

```diff
@@ -412,6 +416,7 @@ protected function formatRoomV4(Room $room, ?Participant $currentParticipant, ?a
 			'listable' => Room::LISTABLE_NONE,
 			'callFlag' => Participant::FLAG_DISCONNECTED,
 			'messageExpiration' => 0,
+			'avatar' => $this->avatarService->getAvatarUrl($room),
```

Comment:
This is the full avatar url, but the documentation more sounds like it's only the `v=` part?